apis/python/src/tiledbsoma/_collection.py

```diff
@@ -425,9 +427,16 @@ def __getitem__(self, key: str) -> CollectionElementType:
             raise KeyError(err_str) from None
         if entry.soma is None:
             from . import _factory  # Delayed binding to resolve circular import.
+            from ._tdb_handles import Wrapper
+
+            wrapper: type[Wrapper[Any | Any | Any]]
```

Comment:
Looks like this `Any` is left over from type troubleshooting?

Reply:
No if I don't put this here then I get an error with the type checker :/